ProductReleaseAnnouncement/product_rel_announcement.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vi: set ft=python :
+"""
+Auto-generate social media posts for Medik8s release announcements by filling in the latest tags and
+release information.
+
+Usage:
+    product_rel_announcement.py --markdown
+    product_rel_announcement.py --html
+    product_rel_announcement.py --html --markdown
+
+Options:
+    -m, --markdown  Write the release announcement file release.md, in Markdown format
+    -w, --html      Write the release announcement file release.html, in HTML format
+"""
+import requests
+import markdown
+from docopt import docopt
+
+
+def main():
+    arguments = docopt(__doc__)
+    NMO = get_latest_version("node-maintenance-operator")
+    NHC = get_latest_version("node-healthcheck-operator")
+    SNR = get_latest_version("self-node-remediation")
+    FAR = get_latest_version("fence-agents-remediation")
+    MDR = get_latest_version("machine-deletion-remediation")
+
+    TEMPLATE = f"""
+Medik8s New Releases
+
+The Medik8s team is thrilled to announce the new releases of ours operators
+
+* Node Maintenance Operator (NMO)    {NMO['tag']} ([release note]({NMO['link']}))
+* Node Healthcheck Operator (NHC)    {NHC['tag']} ([release note]({NHC['link']}))
+* Self Node Remediation (SNR)        {SNR['tag']} ([release note]({SNR['link']}))
+* Fence Agents Remediation (FAR)     {FAR['tag']} ([release note]({FAR['link']}))
+* Machine Deletion Remediation (MDR) {MDR['tag']} ([release note]({MDR['link']}))
+
+see our release notes for the complete list of changes.
```

Comment:
NIT: Capital letter at the beginning of the sentence

```suggestion
See our release notes for the complete list of changes.
```